common/gpu/nvidia/ada-lovelace/default.nix

```diff
@@ -0,0 +1,10 @@
+{lib, config, ...}:
+let
+  nividiaPackage = config.hardware.nvidia.package;
+in
+{
+  imports = [ ../. ];
+
+  # enable the opensorce drivers if the package supports it
+  hardware.nvidia.open = lib.mkOverride 990 (nividiaPackage ? open && nividiaPackage ? firmware);
```

Comment:
What is this priority? Is lib.mkDefault not also sufficient?

Reply:
Ok. I think I answered my question myself: https://github.com/NixOS/nixos-hardware/pull/1093#pullrequestreview-2273752181